.github/workflows/test.yml

```diff
@@ -0,0 +1,68 @@
+name: 🧪 Test
+
+on: 
+  push:
+    branches:
+      - main
+      - dev
+  pull_request_target:
+
+jobs:
+  authorize:
+    environment: 
+      ${{ github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository &&
+      'external' || 'internal' }}
+
+    runs-on: ubuntu-latest
+    steps:
+    - run: echo ✓
+  
+  playwright:
```

Comment:
This is good but we should add the other tests as well (lint, ts and vitest).

Reply:
Okay, should I remove them from Deploy workflow.